.ruby-version

```diff
@@ -1 +1 @@
-3.3.5
+3.2.2
```

Comment:
quick question, was this an intended change?

Reply:
Kinda. :) I thought that specs were failing due to Ruby upgrade and tried to fix this. But it was actually caused by ActiveRecord. I fixed it, so there is no need to revert Ruby now.

Removed, thx!